syntax = "proto3";

import "google/protobuf/any.proto";

message OpaqueContainer {
    google.protobuf.Any opaque = 1;
}